cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -4673,7 +4674,6 @@ function(build_orc)
     set(STOP_BUILD_ON_WARNING
         OFF
         CACHE BOOL "" FORCE)
-
```

Comment:
Could you revert this?
We want to keep this empty line to separate parameter set block and `fetchcontent_makeavailable()`.